arch/archs.yml

```diff
@@ -1,4 +1,21 @@
 archs:
-  - name: empty
-    folder: empty
-    comment: initial file, no arch v2 defined.
+  - name: arc
+    folder: arc
```

Comment:
maybe `path` for consistency with west.yml and binary blobs in module.yml?

Reply:
very good point.
Although it's being considered to rework and remove this file for consistency, now that the similar files for board and soc has been removed.

Reply:
done